Client/src/main/java/edu/sdccd/cisc191/template/Client.java

```diff
@@ -78,14 +78,10 @@ public void start(Stage stage) throws Exception{
         plantsTable.getColumns().addAll( nameColumn, quantityColumn, varietyColumn);
 
         Button addButton = new Button("Add Plant");
-        addButton.setOnAction(e -> {
-            new VegetableGUI(this);
-        });
+        addButton.setOnAction(e -> new VegetableGUI(this));
```

Comment:
Change from Statement lambda to Expression lambda just to make it look more clean.